src/peft/tuners/prompt_tuning/model.py

```diff
@@ -64,7 +64,20 @@ def __init__(self, config, word_embeddings):
 
         total_virtual_tokens = config.num_virtual_tokens * config.num_transformer_submodules
         self.embedding = torch.nn.Embedding(total_virtual_tokens, config.token_dim)
-        if config.prompt_tuning_init == PromptTuningInit.TEXT and not config.inference_mode:
+        if config.prompt_tuning_init == PromptTuningInit.RANDOM_DISCRETE and not config.inference_mode:
+            import numpy as np
```

Comment:
How about using torch directly, instead of creating numpy arrays and later converting them to torch?